release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -1465,6 +1465,11 @@ With this release, additional debugging statements were added to the `ServiceIns
 
 Previously, the self-signed loopback certificate for the Kubernetes API Server expired after one year. With this release, the expiration date of the certificate is extended to three years.
 
+[id="vmw-7-vcf-4-eogs_{context}"]
+=== {vmw-full} 7 and VMware Cloud Foundation 4 end of general support
+
+Broadcom has ended general support for {vmw-full} 7 and VMware Cloud Foundation (VCF) 4. If your existing {product-title} cluster is running on either of these platforms, you must plan to migrate or upgrade your VMware infrastructure to a supported version. {product-title} supports installation on {vmw-short} 8 Update 1 or later, or VCF 5 or later.
```

Comment:
Hi @sslocket Consider using the 7.x and 4.x format since VMware uses the same in their [announcement](https://blogs.vmware.com/cloud-foundation/2024/07/25/announcing-extension-of-vmware-vsphere-7-x-and-vmware-vsan-7-x-general-support-period/).
@mzasepaRedHat - I think we should we include a link to the announcement by VMware. WDYT? Also, do we have an answer for [this](https://github.com/openshift/openshift-docs/pull/99835#issuecomment-3357015546 ) because that would mean a change in the current recommendation to upgrade VCF to 5.x or later.